test/DynamoCoreTests/Logging/FeatureFlagTests.cs

```diff
@@ -76,7 +76,7 @@ public void FeatureFlagsShouldMessageLoggedShouldContainAllLogs()
             StringAssert.Contains("\"TestFlag1\":true", log);
             StringAssert.Contains("\"TestFlag2\":\"I am a string\"", log);
             StringAssert.Contains("\"graphics-primitive-instancing\":true", log);
-            StringAssert.EndsWith("<<<<<Eod>>>>>", log);
+            StringAssert.Contains("<<<<<Eod>>>>>", log);
```

Comment:
what is this about ?

Reply:
@pinzart90 due to the changes you call out below.